src/components/AuthForm.js

```diff
@@ -103,6 +103,36 @@ const baseUISchema = {
 };
 
 const authSchemas = {
+  account: {
```

Comment:
Just me or this `account` schema is basically just the `account` one with its title modified? If so we should rather create a common base and extend it appropriately.